File: app/src/main/java/com/vn/ezlearn/models/Answer.java
package com.vn.ezlearn.models;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 10/10/2017.
 */

public class Answer {
    public static final Integer IS_TRUE = 1;

    public Integer id;
    public Integer question_id;
    public String answer;
    public Integer is_true;
    public String mark;
    public String created_at;
    public String updated_at;
    public Integer status;
    public String weight;
}

File: app/src/main/java/com/vn/ezlearn/models/Answer.kt
package com.vn.ezlearn.models

/**
 * Created by FRAMGIA\nguyen.duc.manh on 10/10/2017.
 */

class Answer {

    var id: Int? = null
    var question_id: Int? = null
    var answer: String? = null
    var is_true: Int = 0
    var mark: String? = null
    var created_at: String? = null
    var updated_at: String? = null
    var status: Int? = null
    var weight: String? = null

    companion object {
        val TRUE = 1
    }
}
